src/back/loader/Base.js

```diff
@@ -72,7 +72,9 @@ BaseLoader.prototype.normalizeStylesheet = function (style) {
 
 BaseLoader.prototype.load = function () {
     this.mml = this.loadFile();
-    this.postprocess();
+    if (this.project.config.parsed_opts.renderer == 'carto') {
```

Comment:
At this point, I'd prefer  use inheritance to deal with this instead of a condition check in the base class. Either add an empty postprocess in the MagnaCarto class or keep it empty in the base class and move the mml dedicated code in the Carto class (maybe this last is better, not sure in the future if a new class would need or not those postprocess steps…).